helm/chart/router/values.yaml

```
@@ -158,9 +158,12 @@ ingress:
 
 # set to true to enable istio's virtualservice
 virtualservice:
-  enabled: false
-  # namespace: ""
-  # gatewayName: ""
+  enabled: true
+  namespace: "default"
```

@marcantoine-bibeau I figured out that the reason our helm check is failing is that it attempts to validate all the components which are enabled in the `values.yaml` file. We don't want `virtualservice` to be enabled by default, since most users of this chart won't be using it. Suggested modification to your PR:
```suggestion
  enabled: false
  # namespace: ""
```

---

Ha! My bad!